sci-mathematics/mathematica/mathematica-13.1.0.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHECKREQS_DISK_BUILD=19G
+inherit check-reqs desktop unpacker xdg
+
+DESCRIPTION="Wolfram Mathematica"
+SRC_URI="
+	doc? ( WLDocs_${PV}_LINUX.sh )
+	Mathematica_${PV}_LINUX.sh
+"
+HOMEPAGE="https://www.wolfram.com/mathematica/"
+
+LICENSE="all-rights-reserved"
+KEYWORDS="-* ~amd64"
+SLOT="0"
+IUSE="cuda +doc ffmpeg R"
```

Comment:
I think the doc flag shouldn't be enable by default. Because with the doc flag enabled we significantly increase the download time.

Reply:
Done